cmd/benchcmp/main.go

```
@@ -0,0 +1,450 @@
+	fs := flag.NewFlagSet("benchcmp", flag.ContinueOnError)
+	fs.SetOutput(stderr)
+	fs.StringVar(&baselinePath, "baseline", "", "baseline benchmark JSON path")
+	fs.StringVar(&currentPath, "current", "", "current benchmark JSON path")
+	fs.StringVar(&summaryOut, "summary-out", "", "optional markdown summary output path")
+	fs.BoolVar(&opts.Strict, "strict", false, "fail when benchmark gates are violated")
+
+	if err := fs.Parse(args); err != nil {
+		return 2
+	}
+	if baselinePath == "" || currentPath == "" {
+		fmt.Fprintln(stderr, "-baseline and -current are required")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)